internal/data/trace.go

```diff
@@ -77,12 +81,20 @@ type TraceID struct {
 	bytes []byte
 }
 
+func (t TraceID) Bytes() []byte {
+	return t.bytes
+}
+
 func TraceIDFromBytes(bytes []byte) TraceID { return TraceID{bytes} }
```

Comment:
Should we also rename this to NewTraceID to make it more idiomatic?

Reply:
Yes, makes sense. changed